src/Sulu/Component/Security/Authorization/AccessControl/AccessControlManager.php

```diff
@@ -82,8 +82,12 @@ public function getPermissions($type, $identifier)
     /**
      * {@inheritdoc}
      */
-    public function getUserPermissions(SecurityCondition $securityCondition, UserInterface $user)
+    public function getUserPermissions(SecurityCondition $securityCondition, $user)
```

Comment:
You can leave the type hint if you make it nullable, can't you? Like `?UserInterface`.